### ISSUE TITLE
Add a License to this repository

### ISSUE THREAD

**Issue by RenaudWasTaken** [jessestuart/tiller-multiarch#16]
Hello!

Thanks a lot for your great work here, I've noticed that there isn't a license to this repository.
By default if you don't have a license, then all rights are reserved and it is not Open Source or Free. You cannot modify or redistribute this code without explicit permission from the copyright holder.

See the following links explaining this a bit more:
- https://choosealicense.com/no-permission/
- https://help.github.com/en/github/building-a-strong-community/adding-a-license-to-a-repository
- https://opensource.stackexchange.com/questions/1720/what-can-i-assume-if-a-publicly-published-project-has-no-license

Redistributing under Apache2 is pretty traditional in the cloud native space and might be what you can look into:
- https://github.com/helm/helm/blob/master/LICENSE
- https://github.com/kubernetes/kubernetes/blob/master/LICENSE

TLDR legal does a good job at explaining that license: https://tldrlegal.com/license/apache-license-2.0-(apache-2.0)

You should use any license you want, just having clarification on how you are distributing this software a good enough resolution of this issue :)
Thanks again!

**Comment by jessestuart** [comment]
Hey @RenaudWasTaken —
Thanks for the super detailed issue! You're right, I've been kind of careless when it comes to licensing my OSS contributions (and I should know better, having formerly working for a patent law company...). 

I've added the Apache 2.0 license to the repo (same as the upstream `helm/helm`).

**Comment by RenaudWasTaken** [comment]
Thanks a lot!
